gae/home.tpl

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="robots" content="noindex">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
```

Comment:
To be consistent with DevSite, let's use `<meta name="viewport" content="width=device-width, initial-scale=1">`